source/ada/lsp-ada_handlers.adb

```diff
@@ -1643,6 +1645,10 @@ package body LSP.Ada_Handlers is
            (Integer (Decl.Sloc_Range.Start_Column), Min_Width => 1)
          & ")");
 
+      if Self.Project_Tree.Root_Project.Is_Aggregate_Project then
+         Location_Text := Location_Text & " in " & C.Id & " project";
```

Comment:
we should write "in project p" rather than "in p project"

Reply:
Done